packages/babel-plugin-transform-async-to-generator/test/fixtures/async-to-generator/statement/actual.js

```diff
@@ -1,5 +1,3 @@
-function normalFunction() {}
```

Comment:
Yep - it was there to test that the async function _wasn't_ hoisted, which is incorrect.

I could put it back (and move it below the async function in the output), but that should be covered by e759b5f835235ea65948f9e35073ab76dec8b5c3.